<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Заголовок</string>
  </property>
  <widget class="QPushButton" name="pushButton_apply">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>247</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_degree">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обжарка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_degree">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ground_or_whatever">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Молотый или нет</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ground_or_whatever">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Объем</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_volume">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_taste">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вкус</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taste">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
